### PR TITLE
unbreak netlify

### PR DESCRIPTION
**Pull request by whimsicallyson**
PR Checklist:

- [ ] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
try to get the netlify preview building again

### CODE REVIEW

**Comment by madkinsz** [comment]
@whimsicallyson did you try ```lines = '''
The first newline is
trimmed in raw strings.
All other whitespace
is preserved.
'''``` ? or with `"` per https://toml.io/en/v1.0.0#string

**Comment by whimsicallyson** [comment]
@madkinsz its a series of bash commands, so not sure if that would work? I just used `&&` to shove them all into one line 😆

File: netlify.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@
   publish = "dist/"
 
   # Default build command.
-  command = "npm run build"
+  command = "echo '@fortawesome:registry=https://npm.fontawesome.com/' > .npmrc && echo '//npm.fontawesome.com/:_authToken=${FA_TOKEN}' >> .npmrc && npm install && npm run build"
```
Comment (madkinsz) [inline netlify.toml:14]: I guess I imagined this for readability -- whatever works though :D

```suggestion
  command = """
      echo '@fortawesome:registry=https://npm.fontawesome.com/' > .npmrc && 
      echo '//npm.fontawesome.com/:_authToken=${FA_TOKEN}' >> .npmrc && 
      npm install && npm run build"
  """
```

Comment (madkinsz) [reply netlify.toml:14]: Weird github syntax highlighting makes this scary

**Review by znicholasbrown — APPROVED** [review]
🥳

**Pull request merged**